README.md

```diff
@@ -148,20 +148,21 @@ This plugin registers the following Fastify hooks:
-[`Span`]: https://github.com/open-telemetry/opentelemetry-js/blob/91612c4d5eb44c79510e1c47399054432295d2fa/packages/opentelemetry-tracing/src/Span.ts#L40
-[`Tracer`]: https://github.com/open-telemetry/opentelemetry-js/blob/91612c4d5eb44c79510e1c47399054432295d2fa/packages/opentelemetry-tracing/src/Tracer.ts#L35
-[`TextMapGetter`]: https://github.com/open-telemetry/opentelemetry-js/blob/91612c4d5eb44c79510e1c47399054432295d2fa/packages/opentelemetry-api/src/context/propagation/TextMapPropagator.ts#L99
-[`defaultTextMapGetter`]: https://github.com/open-telemetry/opentelemetry-js/blob/91612c4d5eb44c79510e1c47399054432295d2fa/packages/opentelemetry-api/src/context/propagation/TextMapPropagator.ts#L116
-[`TextMapSetter`]: https://github.com/open-telemetry/opentelemetry-js/blob/91612c4d5eb44c79510e1c47399054432295d2fa/packages/opentelemetry-api/src/context/propagation/TextMapPropagator.ts#L81
-[`defaultTextMapSetter`]: https://github.com/open-telemetry/opentelemetry-js/blob/91612c4d5eb44c79510e1c47399054432295d2fa/packages/opentelemetry-api/src/context/propagation/TextMapPropagator.ts#L132
+[OpenTelemetry API]: https://github.com/open-telemetry/opentelemetry-js/tree/27a8d4530c56257c191468b073669909bf8892a1/packages/opentelemetry-api
+[`Context`]: https://github.com/open-telemetry/opentelemetry-js/blob/27a8d4530c56257c191468b073669909bf8892a1/packages/opentelemetry-context-base/src/types.ts
+[`Propagation.extract`]: https://github.com/open-telemetry/opentelemetry-js/blob/27a8d4530c56257c191468b073669909bf8892a1/packages/opentelemetry-api/src/api/propagation.ts#L91
+[`Propagation.inject`]: https://github.com/open-telemetry/opentelemetry-js/blob/27a8d4530c56257c191468b073669909bf8892a1/packages/opentelemetry-api/src/api/propagation.ts#L76
+[`Span`]: https://github.com/open-telemetry/opentelemetry-js/blob/27a8d4530c56257c191468b073669909bf8892a1/packages/opentelemetry-api/src/trace/span.ts
+[`Tracer`]: https://github.com/open-telemetry/opentelemetry-js/blob/27a8d4530c56257c191468b073669909bf8892a1/packages/opentelemetry-api/src/trace/tracer.ts
+[`TextMapGetter`]: https://github.com/open-telemetry/opentelemetry-js/blob/27a8d4530c56257c191468b073669909bf8892a1/packages/opentelemetry-api/src/context/propagation/TextMapPropagator.ts#L99
+[`defaultTextMapGetter`]: https://github.com/open-telemetry/opentelemetry-js/blob/27a8d4530c56257c191468b073669909bf8892a1/packages/opentelemetry-api/src/context/propagation/TextMapPropagator.ts#L116
+[`TextMapSetter`]: https://github.com/open-telemetry/opentelemetry-js/blob/27a8d4530c56257c191468b073669909bf8892a1/packages/opentelemetry-api/src/context/propagation/TextMapPropagator.ts#L81
+[`defaultTextMapSetter`]: https://github.com/open-telemetry/opentelemetry-js/blob/27a8d4530c56257c191468b073669909bf8892a1/packages/opentelemetry-api/src/context/propagation/TextMapPropagator.ts#L132
```

Comment:
[suggestion] I can see this being tedious (changing these links each release) should we simple link out to the opentelemetry docs here and leave it to the user to investigate the API? I can see the benefit of this, but maintenance will be a headache down the road I think.

@HW13 Your call!

Reply:
Good call @jkirkpatrick24. I keep forgetting there's an OpenTelemetry `github.io` page with a documented API - I'll point the links at that instead of the code 😎 👍

Reply:
Done ✅   - Although the `Context` link still has to point to the code, as only `@opentelemetry/api` seems to be covered by the `github.io` page and the `Context` interface is defined in `@opentelemetry/context-base`.